srcpkgs/pass-update/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'pass-update'
+pkgname=pass-update
+version=2.1
+revision=1
+archs=noarch
+build_style=gnu-makefile
+depends="pass"
+short_desc="A pass extension providing an easy flow for updating passwords"
+maintainer="Alan Brown <adbrown@rocketmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/roddhjav/pass-update"
+distfiles="https://github.com/roddhjav/pass-update/releases/download/v${version}/pass-update-${version}.tar.gz"
+checksum=2b0022102238e022e7ee945b7622f4c270810cda46508084fcb193582aafaf6f
+
+post_install() {
+	vmkdir usr/share/bash-completion/completions
+	mv ${DESTDIR}/etc/bash_completion.d/* ${DESTDIR}/usr/share/bash-completion/completions
+}
```

Comment:
This block of code could be replaced by this line:
```
make_install_args=BASHCOMPDIR=/usr/share/bash-completion/completions
```

Reply:
Changed this.